CHANGELOG.md

```diff
@@ -26,7 +26,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 
-- fix(openshift): fix remote write proxy - use unprivileged NGINX [#2510][#2510]
+- fix(openshift): fix remote write proxy - use unprivileged NGINX [#2510], [#2510]
```

Comment:
Why this change?

Reply:
Actually, I don't remember exactly the reason now, but I think that some check was failing